input/groups/armor.json

```diff
@@ -765,6 +745,26 @@
         235,
         206
       ],
+      "Crafting Tier: Initiate / Apprentice / Journeyman / Adept - Masquerade / Noble / Gladiator": [
```

Comment:
How about swapping to "Crafting Rarity: <Rarities>" instead? For the first ones it would be "Crafting Rarity: Fine / Masterwork" but all the other ones would be single rarity.

Having "Crafting Rarity: Ascended" would solve the following issue as well.

Reply:
Hmm, it loses a bit of information. I think I'ld prefer to categorise the rare armors as Crafting Tier: Initiate and lose the information they're available in later tiers than drop the crafting tiers entirely.

Reply:
Yeah, just initiate would be fine, you're already unlocking it at a lower level, so anything above it just doesn't really matter, "Crafting Tier: Initiate - Masquerade / Noble / Gladiator" it is then.